README.md

```diff
@@ -840,6 +841,95 @@ public class Handler : IRequiresAsyncInitialization
 public class Container : IContainer<Server> {}
 ```
 
+### `Owned<T>` support
+
+Using `Owned<T>` instead of `T` gives a component the control over when the `T` instance and all its dependencies are released back to the container while maintaining the component’s agnosticism about all the relative lifespans. Injecting `Owned<T>` or delegates returning `Owned<T>` is enabled out of the box, and so are the `AsyncOwned<T>` versions of these. `AsyncOwned<T>` will be required to be used instead of `Owned<T>` when the `T` in question or any of its dependencies require asynchronous disposal.
+
+If you obtain an `Owned<T>`, you access the `T` instance using the `Value` property, and you dispose the `Owned<T>` instance to release the `T` instance back to the container. The container will never dispose the contained `T` instance automatically, not even when the container itself is disposed. It’s now your responsibility to call `Owned<T>.Dispose()` when you’re ready. If you don’t, the `T` instance and all its dependencies will never be cleaned up.
```

Comment:
to release the `T` instance *and it's dependencies* back to the container.

This is important to highlight because otherwise you might think you could dispose `T` directly.